assets/css/disruption_calendar.css

```
@@ -1,23 +1,47 @@
-.fc-event.route-Red {
+.fc-event.status-approved.route-Red {
   background-color: #da291c;
 }
 
-.fc-event.route-Blue {
+.fc-event.status-pending.route-Red {
+  background-color: #f4bfbb;
+}
+
+.fc-event.status-approved.route-Blue {
   background-color: #003da5;
 }
 
-.fc-event.route-Orange {
+.fc-event.status-pending.route-Blue {
+  background-color: #b3c5e4;
+}
+
+.fc-event.status-approved.route-Orange {
   background-color: #ed8b00;
 }
 
-.fc-event.route-Mattapan {
+.fc-event.status-pending.route-Orange {
+  background-color: #f9d6a6;
+}
+
+.fc-event.status-approved.route-Mattapan {
   background-color: #da291c;
 }
 
-.fc-event[class*="route-Green"] {
+.fc-event.status-pending.route-Mattapan {
+  background-color: #f4bfbb;
+}
+
+.fc-event[class*="status-approved route-Green"] {
```

suggestion: the `status-*` part of the class name doesn't need the wild-card matching

```suggestion
.fc-event.status-approved[class*="route-Green"] {
```

(also for the 3 items below)

---

see above comment: I think we both clicked the 🔃 button around the same time :)